### DISCUSSION TITLE
App crashing with no error log

### DISCUSSION THREAD

**Discussion by Carlo8891** [Agontuk/react-native-geolocation-service#327] [Q&A]
Hi, the example app is crashing with no error log as soon as it starts (before it crashes I can only see a blank screen). Has it happened to anyone before?

**Comment by aaculp** [comment]
> Hi, the example app is crashing with no error log as soon as it starts (before it crashes I can only see a blank screen). Has it happened to anyone before?


Was this the error =>  ERROR  Error: Requiring module "node_modules/react-native-geolocation-service/js/index.js", which threw an exception: Invariant Violation: `new NativeEventEmitter()` requires a non-null argument.
